workflows/trigger-lp-build.py

```diff
@@ -174,6 +177,7 @@ def main(argv):
     triggered_build_urls = {}
     valid_arches = ['armhf', 'i386', 'amd64', 'arm64',
                     's390x', 'powerpc', 'ppc64el', 'riscv64']
+    snapcraft_channel = args.get('snapcraft_channel', '')
```

Comment:
we can even determine which channel to use right here - that way we avoid doing it inside the `for` loop, unless I am missing anything

Reply:
this is done now